src/proxyprotocol/address.go

```diff
@@ -0,0 +1,57 @@
+package proxyprotocol
+
+import (
+	"net"
+	"strings"
+
+	proxyproto "github.com/pires/go-proxyproto"
+)
+
+// NewProxyAddr creates an Addr struct from supplied
+// proxyproto.AddressFamilyAndProtocol, net.IP and port as uint16.
+func NewProxyAddr(proto proxyproto.AddressFamilyAndProtocol, addr net.IP, port uint16) net.Addr {
```

Comment:
Ok, I see it's used - I think this should be unexported now though.

Reply:
unexported, but I left the comment in.